dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceConfiguration.java

```diff
@@ -102,6 +102,9 @@
     @NotEmpty
     private String discourseKey;
 
+    @NotEmpty
+    private String discourseCategoryId;
```

Comment:
You could make this an `int` and then you don't have to call `Integer.parseInt()` in EntryResource.java. This would also also cause an invalid string to fail at startup rather than in the middle of EntryResource, where you're not catching a potential NumberFormatException, which might lead to mysterious failures.

We do use `int` for the ElasticSearch config, so there is precedent.